components/drop-zone/index.js

```diff
@@ -62,7 +62,7 @@ class DropZone extends Component {
 
 	render() {
 		const { className, label } = this.props;
-		const { isDraggingOverDocument, isDraggingOverElement, position } = this.state;
+		const { isDraggingOverDocument, isDraggingOverElement, position, isReorderingInProgress } = this.state;
```

Comment:
Never really mind this flag - I am experimenting with a lot of stuff to sort out the conflicts with current drop-zones.